eventdisplay/xml/FairXMLDetectorConf.h

```diff
@@ -0,0 +1,52 @@
+/********************************************************************************
+ * Copyright (C) 2020-2023 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH  *
+ *                                                                              *
+ *              This software is distributed under the terms of the             *
+ *              GNU Lesser General Public Licence (LGPL) version 3,             *
+ *                  copied verbatim in the file "LICENSE"                       *
+ ********************************************************************************/
+
+/*
+ * FairXMLDetectorConf.h
+ *
+ *  Created on: 21 paź 2021
+ *      Author: Daniel Wielanek
+ *		E-mail: daniel.wielanek@gmail.com
+ *		Warsaw University of Technology, Faculty of Physics
+ */
+#ifndef CBMROOT_CORE_EVENTDISPLAY_XML_FAIRXMLDETECTORCONF_H_
+#define CBMROOT_CORE_EVENTDISPLAY_XML_FAIRXMLDETECTORCONF_H_
+
+#include "FairXMLEveConf.h"
+
+#include <Rtypes.h>
+#include <RtypesCore.h>
+#include <TString.h>
+#include <vector>
+
+class TGeoNode;
```

Comment:
Please include the proper include files for ROOT types.

```suggestion
#include <Rtypes.h>
#include <RtypesCore.h>
#include <TGeoNode.h>
#include <TString.h>
#include <vector>
```